# Defining a custom exception
#
# exercise 4.11
import tableformat


def defining_a_custom_exception():
    table_formatter = tableformat.create_formatter('doc')


if __name__ == '__main__':
    defining_a_custom_exception()
